config/webpack.common.js

```diff
@@ -135,6 +138,42 @@ module.exports = {
 			files: '**/*.css',
 		} ),
 
+		new SVGSpritemapPlugin( [
+			'./assets/svg/*.svg'
+		], {
+			output: {
+				filename: 'svg-sprite.svg',
```

Comment:
@colorful-tones can we dump this file into the `/dist/svg/` directory? right now it's just in `/dist`